taxcalc/params.json

```diff
@@ -19,7 +19,7 @@
     },
     "_EITC_c": {
         "long_name": "Maximum earned income credit",
-	    "description": "",
+	"description": "This the maximum amound of earned income credit you're eligible, depending on how many kids you have. ",
```

Comment:
review prose/spelling. Perhaps:
-> This is the maximum amount of earned income credit you're eligible for; it depends on how many kids you have.